src/OpenTelemetry/Trace/TracerProviderBuilderExtensions.cs

```diff
@@ -22,6 +22,23 @@ namespace OpenTelemetry.Trace
 {
     public static class TracerProviderBuilderExtensions
     {
+        /// <summary>
+        /// Sets whether the status of <see cref="System.Diagnostics.Activity"/>
+        /// should be set to <c>Status.Error</c> when it ended abnormally due to an unhandled exception.
+        /// </summary>
+        /// <param name="tracerProviderBuilder">TracerProviderBuilder instance.</param>
+        /// <param name="enabled">Enabled or not. Default value is <c>true</c>.</param>
+        /// <returns>Returns <see cref="TracerProviderBuilder"/> for chaining.</returns>
+        public static TracerProviderBuilder SetErrorStatusOnException(this TracerProviderBuilder tracerProviderBuilder, bool enabled = true)
```

Comment:
@reyang Just curious, what's the use case you were thinking of for adding with `enabled = false`?